enzy_htp/structure/ligand.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, residue_idx: int, residue_name: str, atoms: List[Atom], paren
 
         self.bonds = kwargs.get('bonds', list())  # TODO change this to a more general represetation
         self.conformer_coords = list() # TODO change this to StructureEnsemble (we can leave a reference_var here tho)
+        self.placement_method_ = '' 
```

Comment:
May be make it None for the convention?

The idea for Structure and Ligand is to only have the math model recorded. But I see there is not really a better place to record the source of placement. Would a self.mimo attribute a good idea for these? So that we can distinguish those that should not assumed to exist but just used in some protocols. (similar to what I did in job_manager.ClusterJob)

Reply:
Just add a TODO here is fine for me.